pkg/cloud/cloud.go

```diff
@@ -1073,3 +1087,22 @@ func getVolumeAttachmentsList(volume *ec2.Volume) []string {
 
 	return volumeAttachmentList
 }
+
+// Calculate actual IOPS for a volume and cap it at supported AWS limits.
+// Using requstedIOPSPerGB allows users to create a "fast" storage class
+// (requstedIOPSPerGB = 50 for io1), which can provide the maximum iops
+// that AWS supports for any requestedCapacityGiB.
+func capIOPS(requestedCapacityGiB int64, requstedIOPSPerGB, minTotalIOPS, maxTotalIOPS, maxIOPSPerGB int64) int64 {
+	iops := requestedCapacityGiB * requstedIOPSPerGB
+
+	if iops < minTotalIOPS {
+		iops = minTotalIOPS
+	}
+	if iops > maxTotalIOPS {
```

Comment:
Can we add a log line for the `> max` cases?

Reply:
Added a log into every branch.